resources/virt.go

```diff
@@ -1195,3 +1195,15 @@ func expandHome(p string) (string, error) {
 
 	return p, nil
 }
+
+func NumToAlpha(idx int) string {
+	var result string
+	var mod int = idx % 26
+	var div int = idx / 26
+	if div > 0 {
+		result = NumToAlpha(div - 1) + string(rune(mod+97))
```

Comment:
This can be simplified and inlined, eg

```
return NumToAlpha(div - 1) + string(rune(mod+97))
```

and same for below, dropping the else.